<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>HoMainWindowme</class>
 <widget class="QMainWindow" name="HoMainWindowme">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>517</width>
    <height>638</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">MainWindow</string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">background : #3BB143;</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="Login_btn">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>450</y>
      <width>261</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{
background:#EEEEEE;
border-radius : 5px
}
QPushButton:pressed{
background:#393E46;
border-radius : 5px;
  transform: translateY(4px);
color : #EEEEEE;
}
  

</string>
    </property>
    <property name="text">
     <string>Login</string>
    </property>
    <property name="checkable">
     <bool>false</bool>
    </property>
    <property name="autoDefault">
     <bool>false</bool>
    </property>
    <property name="flat">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QLineEdit" name="Username_edt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>190</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background : #393E46;
padding-left : 20px;
padding-right : 20px;
border-radius : 20px;
border : 4px solid rgb(255,255,255);
color : #fff
}
QLineEdit:focus{
	border : 2px solid #00ADB5;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="placeholderText">
     <string>Username</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>481</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : #eeeeee;</string>
    </property>
    <property name="text">
     <string>LOGIN</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="Username_lb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Password_lb">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>420</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>520</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color : #fffff;</string>
    </property>
    <property name="text">
     <string>Tidak punya akun ?</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>280</x>
      <y>520</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="tabletTracking">
     <bool>false</bool>
    </property>
    <property name="styleSheet">
     <string notr="true">color : #eee;</string>
    </property>
    <property name="text">
     <string>Buat Akun</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Password_edt">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>481</width>
      <height>71</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLineEdit{
background : #393E46;
border : 4px solid rgb(255,255,255);
padding-left : 20px;
padding-right : 20px;
border-radius : 20px;
color : #fff
}
QLineEdit:focus{
	border : 2px solid #00ADB5;
}</string>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="echoMode">
     <enum>QLineEdit::Password</enum>
    </property>
    <property name="placeholderText">
     <string>Password</string>
    </property>
   </widget>
   <widget class="QLabel" name="Username_l_2">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>270</y>
      <width>471</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">color : white;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>40</y>
      <width>161</width>
      <height>151</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../../../../Downloads/3137672.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_4</zorder>
   <zorder>Login_btn</zorder>
   <zorder>Username_edt</zorder>
   <zorder>label</zorder>
   <zorder>Username_lb</zorder>
   <zorder>Password_lb</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>Password_edt</zorder>
   <zorder>Username_l_2</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>517</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
